sphinxcontrib/paverutils.py

```diff
@@ -191,7 +191,6 @@ def run_sphinx(options, *option_sets):
         for (name, value) in getattr(options, 'config_args', {}).items()
     ]
     sphinxopts = [
-        '',
```

Comment:
It seems like we are going to need to check the version of sphinx and optionally insert the extra argument for versions < 1.7.0. Can you update the patch to do that?

Reply:
I don't know right off the top of my head how to get the sphinx version, but I'll find out and update the patch.

Brad

Reply:
There's a version variable in the module:

  $ python
  Python 2.7.12 (default, Dec  4 2017, 14:50:18)
  [GCC 5.4.0 20160609] on linux2
  Type "help", "copyright", "credits" or "license" for more information.
  >>> import sphinx
  >>> sphinx.__version__
  '1.7.0'
  >>>

Reply:
ugh, the rendering of that failed. The variable is "dunder version" so it has double underscore before and after "version"

``sphinx.__version__``

Reply:
OK, that was a little too obvious.  Thanks.